### ISSUE TITLE
Why the RenderHost is null?

### ISSUE THREAD

**Issue by pipited07** [helix-toolkit/helix-toolkit#512]
I want to show the model when the program start but it seems render host always be null.
Here is the code:

```
public MainWindow()
{
view.RenderTechniquesManager = new DefaultRenderTechniquesManager();

view.EffectsManager = new DefaultEffectsManager(view.RenderTechniquesManager);

MeshGeometryModel3D mesh=LoadSTL(path);

DXView.Items.Add(GroupModel);

ModelGeometry.Add(mesh);

GroupModel.Children = ModelGeometry;

mesh.Attach(view.RenderHost);
}
```

I wraped the code in a Button and it did work. The RenderHost is not null anymore. Why is that?

By the way the coordinate in the viewport3dx seems no label X Y Z , even I set the property in xaml
```
CoordinateSystemLabelForeground="Aqua"                 
                        CoordinateSystemLabelX="X"
                        CoordinateSystemLabelY="Y"
                        CoordinateSystemLabelZ="Z"
```
How do I show the label in the viewport?

Hope anyone could help me. Thanks~

**Comment by holance** [comment]
Please see the example implementation, all the components are available
after view port is loaded. Please using binding instead of direct accessing
internal components.

**Comment by pipited07** [comment]
@holance  Actually, I see the example implementation, that is why I wrap the code into a Button.
But  I want to show the model at initial. Is there any way to do that?

**Comment by holance** [comment]
I think you don't need to manually call attach render host. Try to remove it and see if it works

**Comment by pipited07** [comment]
@holance  It work!!!　But I don't know the reason.  The "attach" isn't connect the element and the view port? Then when do I use attach ?

**Comment by holance** [comment]
Once view port is loaded, it will automatically attach all the models. So
never call attach render host manually.

**Comment by pipited07** [comment]
"So never call attach render host manually" this means the beginning right? Because next time load the new model I still have to call attach render. By the way , do you have any idea about my second question?
Thanks for your rapid reply.

**Comment by holance** [comment]
You don't need to call attach even in later, you can check how example works, they don't need to call attach,it has been handled internally. And viewmodel does not accessing to the render host property.

For the second question, I will get back to you later

**Comment by holance** [comment]
Fixed coordinate labels not showing up in #513
